### ISSUE TITLE
Chrome version?

### ISSUE THREAD

**Issue by tomobs** [jomccr/firefox-hjkl#3]
Hi, may u add I chrome version? THANKS!!!

**Comment by jomccr** [comment]
No, firefox only - you should try to build your own alternative with an agent. 

<3 Thanks for checking it out mate.
